packages/ibm-products-styles/src/components/Card/_card.scss

```
@@ -185,7 +199,8 @@ $block-class: #{c4p-settings.$pkg-prefix}--card;
 }
 
 .#{$block-class}__clickable.#{$block-class}--has-slug::before,
-.#{$block-class}__clickable.#{$block-class}--has-ai-label::before {
+.#{$block-class}__clickable.#{$block-class}--has-ai-label::before,
+.#{$block-class}__clickable.#{$block-class}--has-decorator::before {
```

```suggestion
.#{$block-class}__clickable.#{$block-class}--has-decorator::before {
```

I think for `has--decorator`, we don’t want to tie this to the popover styles since this could be separate.

---

Thanks @elycheea,
This is updated.